### ISSUE TITLE
Remove lower PHP versions - clean up build Matrix

### ISSUE THREAD

**Issue by ElectricMaxxx** [sonata-project/SonataDoctrinePhpcrAdminBundle#417]
As we did it in the main CMF bundles, we should have an equal build matrix in here.

**Comment by dbu** [comment]
@Soullivaneuh did something change about the build process? i get lots of "wrong phpunit version for this version of php" on travisci. after #416, i restarted a build that was green last week. now its not anymore... or are we aware of travis-ci having made a mess?

**Comment by greg0ire** [comment]
@dbu phpunit 6.0.0 is out, i suspect it has to do with this.

**Comment by dbu** [comment]
ah right. but smells like travis made a mistake, right? we rely on the
installed phpunit from how i read .travis.yml and the makefile

**Comment by greg0ire** [comment]
I don't think we do, I think we download it. We even download an old version for old php jobs. But the name does not contain the version number, which is bad.

**Comment by greg0ire** [comment]
https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/search?utf8=%E2%9C%93&q=phar

**Comment by greg0ire** [comment]
There are versioned downloads though : https://phar.phpunit.de/

**Comment by dbu** [comment]
ah, hidden in .travis where i seem to not have searched. is phpunit
install in the dev-kit thing of sonata, or should i fix it specifically
in this repository?

**Comment by greg0ire** [comment]
Please try fixing it specifically in this PR to prove that it works. I'm working on a dev-kit PR.

**Comment by greg0ire** [comment]
Maybe we should take this as an occasion to switch to simple-phpunit ? See https://github.com/sonata-project/dev-kit/issues/197 . WDYT?

**Comment by greg0ire** [comment]
As promised: https://github.com/sonata-project/dev-kit/pull/231

**Comment by dbu** [comment]
simple-phpunit would be a good idea imo. actually surprised we see no clash
on yml in sonata.

**Comment by ElectricMaxxx** [comment]
@dbu btw:i also had https://scrutinizer-ci.com/g/sonata-project/SonataDoctrinePhpcrAdminBundle/inspections/ee0847bd-4c6c-42e4-88d5-d42872d65951 the issue with dans glob repository.

**Comment by wouterj** [comment]
@ElectricMaxxx for your information, this bundle is not part of the CMF organization. So we are bound to support decision etc. made by the Sonata Project team. They use some fancy dev-kit tool to generate all config files, so I guess there is not much to do in this PR.

**Comment by greg0ire** [comment]
@wouterj @ElectricMaxxx it should be ok on the stable branch, which should get merged on the master branch this night (right @Soullivaneuh ?). If you are in a hurry, you can do that merge manually.

**Comment by dbu** [comment]
there was a conflict. created a PR with a merge request where i solved the conflict.

**Comment by greg0ire** [comment]
Great job @dbu!
